packages/client/src/runtime/RequestHandler.ts

```diff
@@ -52,13 +52,14 @@ export type Request = {
   headers?: Record<string, string>
   otelParentCtx?: Context
   otelChildCtx?: Context
+  tracingConfig: TracingConfig
```

Comment:
```suggestion
  tracingConfig?: TracingConfig
```

Reply:
I guess I wanted it more for being standard with the other props, and if one day we unit test this (if not already) then we would not need to pass any config.

Reply:
Good point, I did not think about this. Have removed it!